mars/dataframe/groupby/aggregation.py

```diff
@@ -432,9 +619,15 @@ def _perform_shuffle(
         in_df: TileableType,
         out_df: TileableType,
         func_infos: ReductionSteps,
+        pivot_chunk,
     ):
         # Shuffle the aggregation chunk.
-        reduce_chunks = cls._gen_shuffle_chunks(op, in_df, agg_chunks)
+        if op.groupby_params["sort"] and pivot_chunk is not None:
```

Comment:
Maybe check pivot_chunk is OK? line 824 gens pivot_chunk only when some conditions meet.

Reply:
Can you elaborate on this point? 
pivot chunk is None by default. If `op.groupby_params["sort"] and len(in_df.chunks) > 1` only then pivot chunk is not None

Reply:
Yeah, I mean we only need check `if pivot_chunk is not None:`, because if it's not None, `op.groupby_params["sort"]` is definitely True.

Reply:
understood